tensorboard/plugins/projector/tf_projector_plugin/index.js

```diff
@@ -22,6 +22,7 @@ iframe {
   height: 100%;
   margin: 0;
   width: 100%;
+  display: block;
```

Comment:
nit: Not a rule but prefer to a-z sort if not inconvenient. Here and plugins_component.ts